### DISCUSSION TITLE
Angular 18/Slickgrd 8X - Icons are not loading properly

### DISCUSSION THREAD

**Discussion by MvijayM** [ghiscoding/Angular-Slickgrid#1488] [Q&A]
After Angular upgrade to 18 and slickgrid upgrade to 8 MD design icons are not loading properly if I use the bootstrap css file

`@import '@slickgrid-universal/common/dist/styles/css/slickgrid-theme-bootstrap.css';`
![WhatsApp Image 2024-10-22 at 2 24 05 PM](https://github.com/user-attachments/assets/a7d6a80e-7919-41fd-a986-59c93d72c7cc)

Checkbox is displaying as empty box and if I select checkbox it is showing as color filled instead of icon.

**Comment by ghiscoding** [comment]
it works fine for me, however you should follow the Quick Start guide and include it in your `angular.json` under the `styles` section because it doesn't look like you followed that requirement, see [Quick Start](https://ghiscoding.gitbook.io/angular-slickgrid/getting-started/quick-start#id-2.-modify-the-angular.json-and-tsconfig.app.json-files). I can't help more than that since you didn't provide any code repro or anything else.
